app/assets/stylesheets/components/page_specific/_floating_chat.scss

```diff
@@ -35,42 +36,49 @@
     font-size: 1.2rem;
     margin-bottom: 5px;
   }
-  
+
   .chat-box__icon {
-    margin-left: -35px;	
+    margin-left: -35px;
   }
 }
 
-.chat-popup__container {
+.mobile-webchat__container {
   bottom: $baseline-unit*2;
   right: $default-gutter*2;
-  padding: $baseline-unit*2 $baseline-unit*2 $baseline-unit;
-  border-radius: 4px;
-  display: block;
-  transition: all 0.4s;
+  border-radius: .15em;
+  transition: transform 0.5s, bottom 0.4s, width 0.3s cubic-bezier(0.075, 0.82, 0.165, 1);
+  width: 76px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
```

Comment:
Good use of flexbox but I think we need to have a fallback for non-flexbox browsers (IE9 really). Nothing major, maybe just explicit widths. (N.B. I haven't checked what it renders like on such browsers).

Reply:
Even by just removing the flex all the elements look the same with no changes. I'm just using flex here to make sure the children are in a column direction but if the browser doesn't support flex then it'll still going to be looking the same.

Reply:
OK 👍